query/README.md

```diff
@@ -0,0 +1,146 @@
+Keen Query Clients
+===================
+
+[![Build Status](https://travis-ci.org/keenlabs/KeenClient-Java.png?branch=master)](https://travis-ci.org/keenlabs/KeenClient-Java)
+
+The query capabilities within the Java Keen client enable you to send POST queries and receive the results of the queries in a JSON object. For query types, refer to API technical reference: https://keen.io/docs/api/reference/.
```

Comment:
Nit: You could make this a regular markdown link with `[API technical reference](https://keen.io/docs/api/reference/)` (see https://help.github.com/articles/markdown-basics/#links for details).